### DISCUSSION TITLE
更新问题

### DISCUSSION THREAD

**Discussion by guusji** [dotnetcore/FreeSql#1658] [Q&A]
`var accountRepo = _patientRepository.GetRepository<UserAccountEntity>();

var item = new UserAccountEntity { ID = accountId };
accountRepo.Attach(item);
item.Password = password;
item.UpdateTime = DateTime.Now;
var rows = await accountRepo.UpdateAsync(item);
return rows;`
我在更新中，更新了账户密码和更新时间，但是对应记录的另一个字段会变空，希望解答一下

**Comment by 2881099** [comment]
把 SQL 监视打开，查看执行的 SQL
